web/source/osk/visualKeyboard.ts

```diff
@@ -1527,11 +1523,11 @@ namespace com.keyman.osk {
      *  @param  {Object}            PKbd            the keyboard object to be displayed
      *  @param  {string=}           argFormFactor   layout form factor, defaulting to 'desktop'
      *  @param  {(string|number)=}  argLayerId      name or index of layer to show, defaulting to 'default'
-     *  @param  {Object}            host            KeymanWeb's active OSKManager instance 
+     *  @param  {Object}            height          Target height for the rendered keyboard 
```

Comment:
```suggestion
     *  @param  {number}            height          Target height for the rendered keyboard 
```

Although we are trying to move types out of jsdoc and into the function signature I suppose...